recipe-client-addon/test/browser/browser_RecipeRunner.js

```diff
@@ -225,6 +226,7 @@ add_task(async function testStartup() {
 
   // not in dev mode
   await SpecialPowers.pushPrefEnv({set: [["extensions.shield-recipe-client.dev_mode", false]]});
+  await SpecialPowers.pushPrefEnv({set: [["extensions.shield-recipe-client.first_run", false]]});
   RecipeRunner.init();
   ok(!runStub.called, "RecipeRunner.run is not called immediately when not in dev mode");
   ok(addCleanupHandlerStub.called, "A cleanup function is registered when not in dev mode");
```

Comment:
I'm having trouble coming up with a good testing strategy here - for the moment this just makes the current tests pass.

The problem is that shield completes way too early in startup, so listening for a notification or for the pref to change will be far too late to really prove anything the existing test doesn't.

I suppose we could have the shield client execute a recipe that spins the event loop enough times that the test can see a notification? Or perhaps execute a recipe that flips a pref or something else that the test could verify? Any thoughts? :)